src/js/remoteServicesCommunity.js

```diff
@@ -155,4 +155,46 @@ export default {
     },
     allowHostOverride: false,
   },
+  superchat: {
+    name: "superchat",
+    host: "https://app.superchat.de",
+    urlPatterns: [":host:/inbox/:id"],
+    id: (document, service, { id }) => [service.key, id].join("."),
+    description: (document, _service, { id }) => {
+      var subjectsRaw =   document.querySelectorAll("div#__next div div#bodySafeArea  h4.hy-font-semibold.hy-text-sm.hy-mt-6.hy-px-6.hy-cursor-pointer")
+      var subject = id.toString()
+      if(subjectsRaw.length == 1){
+        var subjectRaw = subjectsRaw[0].innerHTML
+        subject = subjectRaw.substring(subjectRaw.indexOf(":")+2)
+      }else{
+        console.debug("subject field not found:")
```

Comment:
Please remove all console outputs.

Reply:
All console outputs are removed on the PR